audnauseum/gui/ui.py

```diff
@@ -19,12 +20,44 @@ def connect_all_inputs(ui, looper: Looper):
     connect_track_control_buttons(ui, looper)
     connect_fx_buttons(ui, looper)
     connect_metronome_buttons(ui, looper)
-    connect_volume_dial(ui, looper)
+    connect_volume(ui, looper)
     initialize_lcd_display(ui, looper)
     connect_load_loop(ui, looper)
     connect_save_loop(ui, looper)
 
 
+# Dictionary for creating unique item names in for-loop modified from source code:
+# https://stackoverflow.com/questions/6181935/how-do-you-create-different-variable-names-while-in-a-loop
+
+def update_track_list(ui, looper: Looper):
+
+    track_list = looper.loop.tracks
+    item = {}
+
+    ui.listWidget.clear()
+
+    index = 1
+    for track in track_list:
+        name = track.file_name.split('/')[-1]
+
+        item["item{0}".format(index)] = QListWidgetItem(name)
+        ui.listWidget.addItem(item["item{0}".format(index)])
+        item["item{0}".format(index)].setText(name)
```

Comment:
This is completely fine as is, only one suggestion might be using f-strings (a new-ish feature in Python since that StackOverflow answer) as slightly more readable -- though only slightly and also that's subjective.
```suggestion
        item[f"item{index}"] = QListWidgetItem(name)
        ui.listWidget.addItem(item[f"item{index}"])
        item[f"item{index}"].setText(name)
```